src/main/java/org/jenkinsci/test/acceptance/po/Node.java

```diff
@@ -44,6 +44,8 @@ protected Node(Injector i, URL url) {
     public abstract String getName();
 
     public void setExecutors(int n) {
+        // On high Res screens the UI redraws, so best wait for numExecutors to be present
+        waitFor(by.path("/numExecutors"));
```

Comment:
@olivergondza Not for every control, most of the time when the containing page is loaded the control is ready to use and no need for wait exists, but in this concrete case the page redraws itself after load to adjust to size, with the result that sometimes 'control("/numExecutors").set(n)' was failing due to 'NoSuchElement' exceptions. Using ẃaitFor' was only a way to give the page time to redraw.